docs/VisualStudioLinux.md

```diff
@@ -0,0 +1,132 @@
+# Using Visual Studio to Develop Enclave Applications for Linux
+
+This walkthrough assumes that you are using Visual Studio on a Windows development
+machine, and want to develop enclaves for Linux.  If you instead want to use Visual
+Studio Code on a development machine running either Windows or Linux, see the
+[VS Code instructions](https://github.com/microsoft/openenclave/blob/feature.new_platforms/new_platforms/vscode-extension/README.md).
+
+## Prerequisites
+
+To develop Linux applications using a Windows development machine, you will need the following.
+
+On the Windows development machine:
+
+- [Visual Studio](https://visualstudio.microsoft.com/downloads/) 2017 or 2019
+  (Community edition, or any other edition)
+- ["Linux development with C++" Visual Studio workload](https://devblogs.microsoft.com/cppblog/linux-development-with-c-in-visual-studio/),
+  installable via Tools -> Get Tools and Features (on VS2017) or Features (on VS2019) ->
+  Workloads -> Other Toolsets -> Linux Development with C++
+- [Open Enclave Wizard - Preview](https://marketplace.visualstudio.com/items?itemName=MS-TCPS.OpenEnclaveSDK-VSIX)
+  Visual Studio extension, v0.5 or later
+
+You will also need a build machine running Ubuntu 16.04 or Ubuntu 18.04.  This can be
+a remote Linux machine, but can simply be a "Generation 2" Linux VM on the Windows
+development machine configured as follows:
```

Comment:
Will you also add instructions for using WSL and simulation mode?

Reply:
Probably in a future changelist (it's not one of the channel 9 variations)